test/conftest.py

```diff
@@ -423,7 +423,7 @@ def weaviate_fixture():
         print("Starting Weaviate servers ...")
         status = subprocess.run(["docker rm haystack_test_weaviate"], shell=True)
         status = subprocess.run(
-            ["docker run -d --name haystack_test_weaviate -p 8080:8080 semitechnologies/weaviate:1.11.0"], shell=True
+            ["docker run -d --name haystack_test_weaviate -p 8080:8080 semitechnologies/weaviate:1.14.0"], shell=True
```

Comment:
Let's upgrade Weaviate in the CI too, before assuming that this upgrade is safe. I'm not 100% confident that the client will handle a new Weaviate version without complaining a bit :smile:

Reply:
My bad, I missed it. I have looked over the weaviate repo, and didn't find any breaking changes that could cause client to fail. I will make this change.

Reply:
@ZanSara I just noticed that I changed on the CI before testing it too:

```
    - name: Run Weaviate
      run: docker run -d -p 8080:8080 --name haystack_test_weaviate --env AUTHENTICATION_ANONYMOUS_ACCESS_ENABLED='true' --env PERSISTENCE_DATA_PATH='/var/lib/weaviate' --env ENABLE_EXPERIMENTAL_BM25='true' --env DISK_USE_READONLY_PERCENTAGE='95' semitechnologies/weaviate:1.14.1
```